### PR TITLE
#226 Add the Create Service popup

### PR DESCRIPTION
**Pull request by josephj**
#### Description
Reusing the Edit Popup for the service creation

#### QA URL
* Create Service - https://demo-roster.efcsydney.org/#/admin/services/new
* Edit Service - https://demo-roster.efcsydney.org/#/admin/services/edit/1

#### Acceptance Criteria
- [ ] Ensure that user can see the basic Create Service popup
- [ ] Ensure that the Edit Service popup still works correctly

### CODE REVIEW

**Comment by josephj** [comment]
Hey @liam-lai @trust2065 @queenahu119 @Den3 @waguei-andrea 

These are the files which is related to your **Edit Service** and **Create Service** tickets. Please have a look.

File: client/src/App.js [hunk 1]
```diff
@@ -51,8 +51,14 @@ export default class App extends Component {
...
-          <Route exact path="/admin" component={Admin} />
-          <Route path="/admin/edit/:id" component={Admin} />
+          <Route exact path="/admin/services" component={AdminServices} />
+          <Route
+            exact
+            path="/admin/services/edit/:id"
+            component={AdminServices}
+          />
+          <Route exact path="/admin/services/new" component={AdminServices} />
+          <Redirect from="/admin" to="/admin/services" />
```
[the remark this answers is not in the document]
Comment (josephj) [reply client/src/App.js]: Currently we use [/admin](https://demo-roster.efcsydney.org/#/admin) to show the services list. In this ticket, I change to [/admin/services](https://demo-roster.efcsydney.org/#/admin/services). Just want to make the URL to be more explicit. Any better solutions to make it easier to understand?

File: client/src/components/StateButton.js [hunk 2]
```diff
@@ -48,4 +57,9 @@ const Button = styled.button.attrs({ type: 'button' })`
   font-size: inherit;
   min-width: 110px;
   padding: 12px;
+  &[disabled] {
```
Comment (trust2065) [inline client/src/components/StateButton.js]: can't find where `disabled` property get used

Comment (josephj) [reply client/src/components/StateButton.js]: I actually mentioned this yesterday. :)  It's for you or Queena who has to disable the button while the required fields are missing.

Remember `<StateButton disabled={isDisabled}/>`

Comment (trust2065) [reply client/src/components/StateButton.js]: Got it, thanks for the explanation.

**Pull request merged**